index.php

```diff
@@ -153,7 +153,7 @@
             </div>
             <div class="third">
                 <h2>Open Source</h2>
-                <p>We respect the rights of our users. All of elementary OS is available for review, scrutiny, modification, and redistribution by anyone. Which improves security and privacy for everyone.</p>
+                <p>We respect the rights of our users. All of elementary OS is available for review, scrutiny, modification, and redistribution by anyone. This improves security and privacy for everyone.</p>
```

Comment:
```suggestion
                <p>We respect the rights of our users. All of elementary OS is available for review, scrutiny, modification, and redistribution by anyone—which improves security and privacy for everyone.</p>
```

Reply:
Yeah, I'd agree. That looks more correct.